src/app/ngx-slimscroll/directives/slimscroll.directive.ts

```diff
@@ -203,6 +219,7 @@ export class SlimScrollDirective implements OnInit, OnDestroy {
 
     this.renderer.setStyle(this.bar, 'height', barHeight);
     this.renderer.setStyle(this.bar, 'display', display);
+    display === 'none' ? this.renderer.addClass(this.wrapper, 'bar-hidden') : this.renderer.removeClass(this.wrapper, 'bar-hidden');
```

Comment:
I cannot see where did you get `bar-hidden` class, is that something you have in your styles?

Reply:
please remove that.